src/main/java/core/basesyntax/service/operations/FruitTransactionCreationServiceImpl.java

```diff
@@ -0,0 +1,40 @@
+package core.basesyntax.service.operations;
+
+import core.basesyntax.dao.FruitDao;
+import core.basesyntax.model.FruitTransaction;
+import java.util.List;
+
+public class FruitTransactionCreationServiceImpl implements FruitTransactionCreationService {
+    private FruitDao dao;
+
+    public FruitTransactionCreationServiceImpl(FruitDao dao) {
+        this.dao = dao;
+    }
+
+    @Override
+    public void createTransactions(List<String[]> data) {
+        FruitTransaction transaction;
+        FruitTransaction.Operation operation;
+        String fruit;
+        int amount;
+        for (String[] parameters : data) {
+            if (parameters[0].equals(FruitTransaction.Operation.BALANCE.getOperation())) {
```

Comment:
This if statement is useless. You have an enum and you can get value using `FruitTransaction.Operation.fromString()`